dup_arm64.go

```diff
@@ -0,0 +1,9 @@
+package panicwrap
```

Comment:
Same as above, the constraints should be [`linux,arm64`](https://github.com/golang/go/commit/ab7e82ef3b773468f008df439568e792eb24d5fd#diff-29f4d5c8f3e1fd58918999c7bfc2bee5R5).